src/Filter.cpp

```diff
@@ -150,7 +149,7 @@ void Filter::subset (std::vector <Task>& output)
       for (auto& task : completed)
       {
         // Set up context for any DOM references.
-        contextTask = task;
+        contextTask = &task;
```

Comment:
I couldn't really come up with an example where this would break (and I guess this behaves same as previous version), but I wondered here if we should store and restore the previous value of `contextTask` pointer before and after evaluating the DOM reference.

Reply:
Ah, I see addressing this might be already part of #2683 ?